client/tests/stubs.js

```diff
@@ -62,7 +62,7 @@ class PageChooserModal {}
 global.PageChooserModal = PageChooserModal;
 
 /** Mock window.scrollTo as not provided via JSDom */
-window.scrollTo = jest.fn();
+window.scrollTo = () => {};
```

Comment:
I couldn't use Storybook until I changed this, the console shows "jest is undefined" or something like that because this stubs file is imported in some stories. Seems to be a regression in #10598.